paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/TransformSpecToElement.kt

```diff
@@ -37,29 +38,34 @@ private fun transform(
     focusRequesterCount: FocusRequesterCount
 ): FormElement.SectionElement {
 
-    val fieldElement = when (spec.field) {
-        SectionFieldSpec.Email -> transform(
-            spec.field as SectionFieldSpec.Email,
-            focusRequesterCount
-        )
-        SectionFieldSpec.Name -> transform(
-            spec.field as SectionFieldSpec.Name,
-            focusRequesterCount
-        )
-        is SectionFieldSpec.Country -> transform(
-            spec.field
-        )
-        SectionFieldSpec.IdealBank -> transform(
-            spec.field as SectionFieldSpec.IdealBank
-        )
+    val fieldElements = spec.fields.map {
+        when (it) {
+            SectionFieldSpec.Email -> transform(
+                it as SectionFieldSpec.Email,
```

Comment:
This cast shouldn't be necessary with `when`. Maybe you need to use a named variable instead of `it`?

Reply:
ah, need to use `is`